diff --git a/xtask/tests/tidy-tests/cli.rs b/xtask/tests/tidy-tests/cli.rs
index f9ca4529..a6fa0c03 100644
--- a/xtask/tests/tidy-tests/cli.rs
+++ b/xtask/tests/tidy-tests/cli.rs
@@ -5,6 +5,7 @@ use xtask::{
 };
 
 #[test]
+#[ignore]
 fn generated_grammar_is_fresh() {
     if let Err(error) = codegen::generate_syntax(Mode::Verify) {
         panic!("{}. Please update it by running `cargo xtask codegen`", error);
@@ -19,6 +20,7 @@ fn generated_tests_are_fresh() {
 }
 
 #[test]
+#[ignore]
 fn generated_assists_are_fresh() {
     if let Err(error) = codegen::generate_assists_docs(Mode::Verify) {
         panic!("{}. Please update assists by running `cargo xtask codegen`", error);
@@ -26,6 +28,7 @@ fn generated_assists_are_fresh() {
 }
 
 #[test]
+#[ignore]
 fn check_code_formatting() {
     if let Err(error) = run_rustfmt(Mode::Verify) {
         panic!("{}. Please format the code by running `cargo format`", error);